Add abstract (#276)
w3c/distributed-tracing: spec/00-abstract.md

> ## Abstract

Reviewer: no need. Header should be added by the reSpec

Author: Dropped it.

> This specification defines a standard format to propagate context

Reviewer: format -> headers and value format

Reviewer: ```suggestion
This specification defines a standard headers and value format to propagate context
```

> information for distributed traces. The specification standardizes how context

Reviewer: ```suggestion
information that enables distributed tracing scenarios. The specification standardizes how context
```

Author: Don't know how this helps to add clarity.

Reviewer: this information enables distributed tracing scenarios, but also can be used for other purposes. That was my main point here

Reviewer: I moved comment to the latest version so it will not be lost and easier for you to accept/reject it

> information for distributed traces. The specification standardizes how context

Reviewer: ```suggestion
information for distributed traces. Context information identifies individual requests in a distributed system and also provides means to add and propagate information that is specific for a tracing vendor.
```

Reviewer: this comment tell explicitly that this spec is a first spec the identifies individual request

Reviewer: I believe it's important to note that this is the first spec that defines request identification mechanism.

Reviewer: Pushed a commit that resolves it

information is sent between services and also defines a means to add

> provider-specific context information. 

Reviewer: provider -> tracing vendor?

Author: not everyone is a vendor that's why I used provider.
